spec/requests/events/events_spec.rb

```diff
@@ -3,13 +3,20 @@
 require 'rails_helper'
 
 RSpec.describe EventsController do
+  let(:user) { create(:user) }
+  let(:headers) { { 'Accept' => 'application/json', 'Content-Type' => 'application/json' } }
+  let(:auth_headers) { Devise::JWT::TestHelpers.auth_headers(headers, user) }
   let(:endpoint) { '/events' }
   let(:event) { create(:event) }
   let(:invalid_event) { create(:invalid_event) }
+  let(:custom_headers) do
```

Comment:
the same applies here

Reply:
Which file would you suggest we put the helper? I was thinking to avoid users moving to a different file to looks for methods which can be implemented as one liners?